src/main/java/com/tweetservice/controller/TweetController.java

```diff
@@ -0,0 +1,20 @@
+package com.tweetservice.controller;
+
+import com.tweetservice.model.Tweet;
+import com.tweetservice.service.TweetService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class TweetController {
+    @Autowired
+    TweetService tweetService;
```

Comment:
field injections are bad. Use constructor injection in pair with final properties